headers.go

```diff
@@ -97,11 +98,17 @@ func (h *ProtectedHeader) UnmarshalCBOR(data []byte) error {
 	return nil
 }
 
-// SetAlgorithm sets the algorithm value to the algorithm header.
+// SetAlgorithm sets the algorithm value of the protected header.
 func (h ProtectedHeader) SetAlgorithm(alg Algorithm) {
 	h[HeaderLabelAlgorithm] = alg
 }
 
+// SetCWTClaims sets the CWT Claims value of the protected header.
+func (h ProtectedHeader) SetCWTClaims(claims CWTClaims) {
+	// TODO: validate claims, for example ensuring that 1 and 2 are tstr, not bstr
```

Comment:
It could be argued this is essential to do, before merging this.

Reply:
Proposing to track as a separate issue, rather than block this PR. See #186